web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -748,14 +802,17 @@ class AppConfig extends Component<Props, State> {
                         )}
                         {!savingConfig && (
                           <div className="ConfigError--wrapper flex-column alignItems--flexStart">
-                            {configError && (
-                              <span className="u-textColor--error u-marginBottom--20 u-fontWeight--bold">
-                                {configError}
-                              </span>
-                            )}
+                            {configError ||
+                              (this.state.showValidationError && (
+                                <span className="u-textColor--error tw-mb-2 tw-text-xs">
+                                  {configError ||
+                                    "Error detected. Please use config nav to the left to locate and fix issues."}
+                                </span>
+                              ))}
```

Comment:
show error above button. 

This error already exists but I tweaked it to show the validation error if there's no configError. I also reduced the size so that it shows up like the rest of the errors.